app.py

```diff
@@ -134,8 +140,12 @@ def feedback(name=None):
         "sv": _("Swedish"),
     }
 
+    form.manufacturer.data = request.args.get("manufacturer")
```

Comment:
You might want to change it in Android because it's already just "manufacturer" in iOS and here

Reply:
Those are still unmerged PRs, where as the Android `device_manufacturer` is already in production and will require a new release to change (and the old release will still keep sending `device_manufacturer` here for a while). But it can be changed there, too.